model/src/datetime/display.rs

```diff
@@ -1,16 +1,8 @@
 //! Display implementation for formatting a [`Timestamp`].
 
-use crate::datetime::constant::{
-    self, DAYS_PER_YEAR, EPOCH_UNIX, MICROSECONDS_PER_SECOND, MINUTES_PER_HOUR, SECONDS_PER_HOUR,
-    YEARS_PER_LEAP, YEAR_MAX,
-};
-
-use super::{
-    constant::{Month, SECONDS_PER_DAY},
-    Timestamp,
-};
+use super::Timestamp;
 use serde::{Serialize, Serializer};
-use std::fmt::{Display, Error as FmtError, Formatter, Result as FmtResult};
+use std::fmt::{Display, Formatter, Result as FmtResult};
 
 /// Display implementation to format a [`Timestamp`] in an ISO 8601 format.
 ///
```

Comment:
"Timestamps up to and including the year 2038 are supported". No longer true, up to year 9999 is supported